implementation/domain/event-api/jbb-board-event-api/src/main/java/org/jbb/board/event/ForumCreatedEvent.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2017 the original author or authors.
+ *
+ * This file is part of jBB Application Project.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  You may obtain a copy of the License at
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package org.jbb.board.event;
+
+import org.apache.commons.lang3.Validate;
+import org.jbb.lib.eventbus.JbbEvent;
+
+import lombok.Getter;
+import lombok.ToString;
+
+@ToString
+public class ForumCreatedEvent extends JbbEvent {
+    @Getter
+    private Long forumId;
+
+    public ForumCreatedEvent(Long forumId) {
+        Validate.notNull(forumId);
```

Comment:
Yeah, I think we can just annotate event fields just like we doing at database entities fields with validation constraints. I opened issue for that: http://www.hostedredmine.com/issues/660912